libs/common/src/vm-query/dtos/vm-query.arguments.ts

```diff
@@ -0,0 +1,19 @@
+export class VmQueryArguments {
+  constructor(init?: Partial<VmQueryArguments>) {
+    Object.assign(this, init);
+  }
+
+  contractAddress: string = '';
+  functionName: string = '';
+  callerAddress: string = '';
+  args?: string[];
+
+  static toGatewayBody(args: VmQueryArguments): any {
```

Comment:
You can simplify the `toGatewayBody` method. You can directly destructure args in the method for clarity.

you can define something like:


```
const { contractAddress, functionName, callerAddress, args: functionArgs } = args;
    return {
      scAddress: contractAddress,
      funcName: functionName,
      caller: callerAddress,
      args: functionArgs,
    };
```

Reply:
done